src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-report.component.ts

```
@@ -67,65 +43,97 @@
                         ?.split('\n')
                         .slice(entry.startLine - 1, entry.startLine + entry.lineCount - 1);
                 }
             })
             .filter((line) => line && line.trim().length !== 0).length;
+    });
 
-        this.removedLineCount = this.entries
+    readonly removedLineCount = computed(() => {
+        return this.sortedEntries()
             .flatMap((entry) => {
                 if (entry && entry.previousFilePath && entry.previousStartLine && entry.previousLineCount) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)